drivers/media/i2c/ov5647.c

```diff
@@ -479,38 +528,126 @@ static int ov5647_enum_mbus_code(struct v4l2_subdev *sd,
 				struct v4l2_subdev_pad_config *cfg,
 				struct v4l2_subdev_mbus_code_enum *code)
 {
-	if (code->index > 0)
+	if (code->index == 0 && ARRAY_SIZE(supported_modes_8bit))
+		code->code = MEDIA_BUS_FMT_SBGGR8_1X8;
+	else if (code->index == 0 && ARRAY_SIZE(supported_modes_8bit) == 0 &&
```

Comment:
Does checkpatch.pl not want this to be ```!code->index && !ARRAY_SIZE(support_modes_8bit)```? Similarly in the other lines.

Reply:
Is it only pointers that checkpatch complains over then? I'm suspecting so having looked at imx219 further.